skyflow/vault/_client.py

```diff
@@ -83,6 +101,12 @@ def detokenize(self, records):
             return result
 
     def get(self, records):
+        '''
+        Returns records by Skyflow IDs or column values.
+
+        :param records: Takes a Dictionary that contains either an array of Skyflow IDs or a unique column name and values.
```

Comment:
Dictionary that contains either an array of Skyflow IDs or a the name of a unique column and associated values.